tests/render/test_column_section_renderer.py

```diff
@@ -1029,7 +993,6 @@ def test_ExpectationSuiteColumnSectionRenderer_expectation_with_string_list_meta
     result_json = (
         ExpectationSuiteColumnSectionRenderer().render(expectations).to_json_dict()
     )
-    print(result_json)
```

Comment:
Why are you removing these print statements as part of this PR?

Reply:
They are outputted to the console when `pytest` is run, and are tested using `assert` statements directly afterwards. It was a part of some clean-up that I was attempting to do as part of this PR.

Do you think it would be better to keep them in?

Reply:
@Shinnnyshinshin 
pytest captures stdout/stderr by default, and only shows it on a test failure. This is super useful for debugging failing tests.
If you are experiencing something else then something is wrong with the configuration.

Where did you see this extra console output when running the tests? Locally or in CI?

https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html#default-stdout-stderr-stdin-capturing-behaviour